phpstan.neon

```diff
@@ -0,0 +1,30 @@
+# Configuration file for PHPStan static code checking, see https://phpstan.org .
+includes:
+  - phar://phpstan.phar/conf/bleedingEdge.neon
+
+parameters:
+
+  level: 1
```

Comment:
@josebc I think we are able to handle level 2 checks 
https://phpstan.org/user-guide/rule-levels

Reply:
@josebc would you like to have `level: 2` or keep it `level: 1` at this time ?